scheduler/test/cook/test/mesos/group.clj

```diff
@@ -0,0 +1,74 @@
+                                     :start-time (tc/to-date (t/ago (t/hours 3)))
+                                     :end-time (tc/to-date (t/ago (t/hours 2))))
+            job-b (create-dummy-job conn :group group-ent-id)
+            _ (create-dummy-instance conn job-b :instance-status :instance.status/running
+                                     :start-time (tc/to-date (t/ago (t/minutes 30))))
+            group-ent (d/entity (d/db conn) group-ent-id)]
+        (is (nil? (seq (group/find-stragglers group-ent))))))
+    (testing "quantile deviation straggler-handling, no stragglers"
+      (let [straggler-handling {:straggler-handling/type :straggler-handling.type/quantile-deviation
+                                :straggler-handling/parameters 
+                                {:straggler-handling.quantile-deviation/quantile 0.5
+                                 :straggler-handling.quantile-deviation/multiplier 2.0}}
+            group-ent-id (create-dummy-group conn :straggler-handling straggler-handling)
+            job-a (create-dummy-job conn :group group-ent-id)
+            _ (create-dummy-instance conn job-a :instance-status :instance.status/success
+                                     :start-time (tc/to-date (t/ago (t/hours 3)))
+                                     :end-time (tc/to-date (t/ago (t/hours 2))))
+            job-b (create-dummy-job conn :group group-ent-id)
+            _ (create-dummy-instance conn job-b :instance-status :instance.status/running
+                                     :start-time (tc/to-date (t/ago (t/minutes 90))))
+            group-ent (d/entity (d/db conn) group-ent-id)]
+        (is (nil? (seq (group/find-stragglers group-ent))))))
+    (testing "quantile deviation straggler-handling, stragglers found"
+      (let [straggler-handling {:straggler-handling/type :straggler-handling.type/quantile-deviation
+                                :straggler-handling/parameters 
+                                {:straggler-handling.quantile-deviation/quantile 0.5
+                                 :straggler-handling.quantile-deviation/multiplier 2.0}}
+            group-ent-id (create-dummy-group conn :straggler-handling straggler-handling)
+            job-a (create-dummy-job conn :group group-ent-id)
+            _ (create-dummy-instance conn job-a :instance-status :instance.status/success
+                                     :start-time (tc/to-date (t/ago (t/hours 3)))
+                                     :end-time (tc/to-date (t/ago (t/hours 2))))
+            job-b (create-dummy-job conn :group group-ent-id)
+            straggler (create-dummy-instance conn job-b :instance-status :instance.status/running
+                                             :start-time (tc/to-date (t/ago (t/minutes 190))))
+            db (d/db conn)
+            group-ent (d/entity db group-ent-id)
+            straggler-task (d/entity db straggler)]
+        (is (= [straggler-task](group/find-stragglers group-ent)))))
```

Comment:
Sorry, I meant the 'stragglers found' test. I feel like having a test with only two jobs doesn't really test the 'quantile' part of detection. I would rest easier if the test (or an additional test) worked with a quantile-index that is not in the extremes of the list of jobs, for example, quantile 0.75 in a set of 10 jobs.